CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 Older Logs can be found in `docs/RELEASE-NOTES-*`.
 
+## [3.7.13.3] - 2018-05-31
+
+### Fixed
+#### gr-blocks
+- using int32 limits from `<cstdint>` was bad judgement, as that only became available with C++11. As the 3.7 series necessitates support for dated compilers, inclusion of `stdint.h` or `cstdint` was made conditional on the C++ version.
```

Comment:
I'd remove the opinion from this and just state the problem and resolution.
"Selection of `stdint.h` or `cstdint` was made conditional on the C++ version as 3.7 releases are required to support compilers lacking C++11"